hummingbot/strategy/__utils__/trailing_indicators/historical_volatility.py

```diff
@@ -0,0 +1,26 @@
+from .base_trailing_indicator import BaseTrailingIndicator
+import numpy as np
+from decimal import Decimal
+
+s_decimal_0 = Decimal("0")
+
+
+class HistoricalVolatilityIndicator(BaseTrailingIndicator):
+    def __init__(self, sampling_length: int = 30, processing_length: int = 15):
+        super().__init__(sampling_length, processing_length)
+        self._previous_sample = s_decimal_0
+
+    def add_sample(self, value: float):
```

Comment:
add_sample should be handled on the strategy file, not inside the indicator calculation, as it happens with the avellaneda strategy and the `AverageVolatilityIndicator`

We want to be possible to use different inputs to populate the price buffer.

Reply:
this simply override the add_sample from the base class.